cli/enry/main.go

```diff
@@ -136,6 +139,50 @@ func printPercents(out map[string][]string, buff *bytes.Buffer) {
 	}
 }
 
+func printFile(file string, buff *bytes.Buffer) {
```

Comment:
the name and the input parameters are a little weird because I try to keep coherence with the other functions that prints the json etc.